### ISSUE TITLE
Allow Creation of Personal Trees and Restore Single Pane Tree View Feature

### ISSUE THREAD

**Issue by browniebraun** [Cacti/cacti#2183]
It is absolutely confusing if you have a huge geographical tree and your customers are searching for all hosts located somewhere. In this case current tree search returns the location, but hides all related hosts. 
I suggest to set "show_only_matches" to false for jstree.

**Comment by cigamit** [comment]
Nice suggestion.  Maybe a user level setting?  Feel free to create a pull request on this one.

**Comment by browniebraun** [comment]
Nope, in regards to user level settings I would prefer something like a "Personal tree" that every user can define for its own as well as global trees that can be opened / modified based on group/user rules, (or generated automatically based on an external DB).
And a drop down field below the tree search input field to select the tree I would like to see.
This "package" is something that I would call a feature request. ;)

Regards
-Andi

**Comment by cigamit** [comment]
Well, though there is not a feature request open on this, it was always the goal to allow users to create trees similar to quicktree, but once we did the first release of Cacti 1.x, I got distracted with everything else.  So, we need to get 1.2 released, and then take some more time for some of the dozens of feature requests that are out there.

**Comment by cigamit** [comment]
I've changed the name a bit.  Users have already complained that they want the Single pane tree view feature restored.

**Comment by yuqi33** [comment]
> I've changed the name a bit. Users have already complained that they want the Single pane tree view feature restored.

I recently upgraded from 0.8.8b to 1.2.7 and found that the single pane view has been cancelled，because my tree structure has many branches. Now I have to open each branch one by one to see the graphics I want. Inconvenient, I hope to restore the single pane view in the 0.8.8 version as soon as possible, thank you very much!

**Comment by TheWitness** [comment]
This is on the roadmap.  I have been given permission to contribute again.  I think I will take up this project soon.

**Comment by bernisys** [comment]
When considering to get back thinking about this feature, please also take into consideration to create a report out of a personal tree. This would speed up report generation by a huge amount!

We're using Howie's great QuickTree add-on at the moment but sadly it does not offer this feature.
